packages/create-plugin/templates/_partials/frontend-getting-started.md

```diff
@@ -6,14 +6,10 @@
    yarn install
    ```
 
-2. Build plugin in development mode or run in watch mode
+2. Build plugin in development mode and run in watch mode
```

Comment:
Really good of you to help update our docs! 👏 

We recently introduced a documentation website which we'd like to (eventually) become the source of truth for all the info contained in these scattered readme docs. Would you mind syncing this branch with main and updating the [frontend docs file there](https://github.com/grafana/plugin-tools/blob/main/docusaurus/docs/frontend.md) too please?